meson.build

```diff
@@ -0,0 +1,81 @@
+project('D-YAML', 'd',
+    meson_version: '>=0.40.0',
+    subproject_dir: 'contrib/subprojects'
+)
+
+project_version      = '0.5.4'
+project_soversion    = '0'
+
+src_dir = include_directories('source/')
+pkgc = import('pkgconfig')
+
+dyaml_src = [
+    'source/dyaml/composer.d',
```

Comment:
Would you mind sorting the list? That should make adding/removing files easier to track.

```Python
dyaml_src = [
    './source/dyaml/all.d',
    './source/dyaml/anchor.d',
    './source/dyaml/composer.d',
    './source/dyaml/constructor.d',
    './source/dyaml/dumper.d',
    './source/dyaml/emitter.d',
    './source/dyaml/encoding.d',
    './source/dyaml/escapes.d',
    './source/dyaml/event.d',
    './source/dyaml/exception.d',
    './source/dyaml/fastcharsearch.d',
    './source/dyaml/flags.d',
    './source/dyaml/hacks.d',
    './source/dyaml/linebreak.d',
    './source/dyaml/loader.d',
    './source/dyaml/node.d',
    './source/dyaml/nogcutil.d',
    './source/dyaml/parser.d',
    './source/dyaml/queue.d',
    './source/dyaml/reader.d',
    './source/dyaml/representer.d',
    './source/dyaml/resolver.d',
    './source/dyaml/scanner.d',
    './source/dyaml/serializer.d',
    './source/dyaml/stream.d',
    './source/dyaml/style.d',
    './source/dyaml/tag.d',
    './source/dyaml/tagdirective.d',
    './source/dyaml/testcommon.d',
    './source/dyaml/testcompare.d',
    './source/dyaml/testconstructor.d',
    './source/dyaml/testemitter.d',
    './source/dyaml/testerrors.d',
    './source/dyaml/testinputoutput.d',
    './source/dyaml/testreader.d',
    './source/dyaml/testrepresenter.d',
    './source/dyaml/testresolver.d',
    './source/dyaml/testtokens.d',
    './source/dyaml/token.d',
    './source/dyaml/unused.d',
    './source/dyaml/zerostring.d',
    './source/yaml.d'
]
```